CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Description of the upcoming release here.
 
 ### Changed
 
+- [#1614](https://github.com/FuelLabs/fuel-core/pull/#1614): Use default consensus key regardless of trigger mode.
```

Comment:
It is a breaking change, because affects CLI, could you elaborate more how the new usage looks like, please?=)